src/components/ListResources.vue

```diff
@@ -1,5 +1,16 @@
 <template>
   <oc-table-simple data-testid="list-resources-table">
+    <oc-tr>
+      <oc-th>
+        <oc-checkbox
+          class="file-picker-resource-checkbox oc-margin-small-left"
+          :hide-label="true"
+          :value="selectAllChecked"
```

Comment:
Now you can use `areAllResourcesSelected` here and completely remove `selectAllChecked` 🙂